publisher-sdk/src/main/java/com/criteo/publisher/BidResponse.java

```diff
@@ -19,30 +19,39 @@
 import androidx.annotation.Keep;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import com.criteo.publisher.model.AbstractTokenValue;
-import com.criteo.publisher.model.AdUnit;
+import com.criteo.publisher.model.CdbResponseSlot;
+import com.criteo.publisher.model.nativeads.NativeAssets;
 import com.criteo.publisher.util.AdUnitType;
-import com.criteo.publisher.util.ObjectUtils;
+import kotlin.jvm.functions.Function1;
 
 public class BidResponse {
 
   @SuppressWarnings("ConstantConditions")
-  public static final BidResponse NO_BID = new BidResponse(0, false, null);
+  public static final BidResponse NO_BID = new BidResponse(0, false, null, null, null);
 
   private final double price;
 
   private final boolean valid;
 
   private final AdUnitType adUnitType;
 
-  protected BidResponse(
+  private final Clock clock;
+
+  @Nullable
+  private CdbResponseSlot slot;
+
+  BidResponse(
       double price,
       boolean valid,
```

Comment:
Could you refresh my memory on what `valid` is for? Why would a publisher pass `false` here?

Reply:
This was used to tell to publisher if the bid response was successful or not. But 609f946 is replacing the `NO_BID` object by `null`. This boolean is then removed in 609f946 too.